frontend/src/Components/Glyphs.js

```diff
@@ -0,0 +1,142 @@
+import React, { Component } from "react";
+import * as d3 from "d3";
+import "./Glyphs.css";
+
+const availableColor = '#0D77AC';
+const availableColorLighter = '#C1D6E4';
+const unavailableColor = '#d8d8d8';
+const bestPractices = [
+		'no_fee',
+		'no_notary_required',
+		'no_witness_required',
+		'office_contact',
+		'electronic_request'
+	];
+
+class Glyphs extends Component {
+
+	constructor(props) {
+        super(props);
+        this.state = {
+            states:props.states
+        };
+    }
+
+	componentDidMount() {
+        this.initVis();
+    }
+
+    initVis() {
```

Comment:
would you mind adding a comment above your functions to explain what each thing does :)